zeppelin-interpreter/pom.xml

```diff
@@ -218,6 +219,12 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.antlr</groupId>
+      <artifactId>antlr4</artifactId>
```

Comment:
I think that the antlr4-runtime is sufficient. Why is the antlr compiler needed?